en-US/firefox/welcome/page4.lang

```diff
@@ -0,0 +1,71 @@
+## NOTE: demo page available at https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/welcome/4/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/welcome/4/
+
+
+# HTML page title
+;Download the Firefox Browser on your Mobile for iOS and Android
+Download the Firefox Browser on your Mobile for iOS and Android
+
+
+# the strong tag here is to underline Firefox.
+;Wallet. Keys. Phone. <strong>Firefox.</strong>
+Wallet. Keys. Phone. <strong>Firefox.</strong>
+
+
+;Take privacy with you on every device — and leave the data trackers behind.
+Take privacy with you on every device — and leave the data trackers behind.
+
+
+;Get the Firefox App
+Get the Firefox App
+
+
+;Get Firefox on your Phone
+Get Firefox on your Phone
+
+
+;Send the download link right to your phone or email.
+Send the download link right to your phone or email.
+
+
+;Enter your phone number or email
+Enter your phone number or email
+
+
+# This string is reused from firefox/whatsnew
+;Download Firefox for your smartphone and tablet.
+Download Firefox for your smartphone and tablet.
+
+
+;“Firefox: Private, Safe Browser” on iOS or Android.
+“Firefox: Private, Safe Browser” on iOS or Android.
+
+
+# "off your trail" is an expression for not being followed around.
+;Get data trackers off your trail
+Get data trackers off your trail
+
+
+;Enhanced Tracking Protection <a href="%(privacy)s">blocks 2000+ trackers</a> from chasing you around the web.
```

Comment:
The URL points to page only localized in a few languages. Should comment that the page is only available in a few languages (currently not an opt-in page).

Reply:
Will link to the [enhanced tracking protection SUMO page](https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop) for other locales. :+1: thanks, Peiying!